example/ck_tile/17_grouped_gemm/quant_grouped_gemm.cpp

```diff
@@ -75,7 +76,7 @@ float grouped_gemm_tileloop(const ck_tile::stream_config& s,
                                                AccDataType,
                                                GemmShape,
                                                GemmUniversalTraits,
-                                               128>, // QuantGroupSize
+                                               QuantGroupSize>, // QuantGroupSize
```

Comment:
Is the trailing comment redundant?
```suggestion
                                               QuantGroupSize>,
```

Reply:
Updated.